### PR TITLE
Update README with Hardhat instructions

### PR DESCRIPTION
**Pull request by kerzhner**
Also includes formatting changes from format-on-save locally

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,40 +1,57 @@
...
+docker network connect testground-control hardhat
+```
+
+### Instructions
 
 Install testground and build it:
+
 ```sh
-git clone https://github.com/testground/testground.git
+git clone https://github.com/statechannels/testground.git
```
Comment (geoknee) [inline readme.md:24]: Is it worth highlighting that this is our fork?

**Review by geoknee — APPROVED** [review]
Thanks @kerzhner !

Do you think it is worth including instructions here for launching tasks on our cloud runner? (We could obscure the IP address).

**Comment by kerzhner** [comment]
> Do you think it is worth including instructions here for launching tasks on our cloud runner? (We could obscure the IP address).

I am not sure. I think these instructions would just highlight the `--endpoint` testground flag. What do you think?

**Comment by geoknee** [comment]
> I am not sure. I think these instructions would just highlight the `--endpoint` testground flag. What do you think?

Perhaps we can just include a link to the page on notion which has the instructions?

**Pull request merged**